Dockerfile

```diff
@@ -112,20 +95,26 @@ RUN docker-php-ext-install bcmath xsl
 
 RUN apt-get install -y mariadb-client
 
-RUN composer global require hirak/prestissimo
+# Install Drupal.
+WORKDIR /var/www
+
+RUN rm -rf html
+RUN composer create-project drupal/legacy-project:^8.9 html
+
+WORKDIR /var/www/html
+
+RUN composer require --update-with-dependencies drupal/core-dev:^8.9 wikimedia/composer-merge-plugin:^2.0
+RUN chown -R www-data:www-data sites modules themes
 
 # Cache currently used libraries to improve build times. We need to force
 # discarding changes as Drupal removes test code in /vendor.
-RUN cd /var/www/html \
-  && cp composer.json composer.json.original \
+RUN cp composer.json composer.json.original \
   && cp composer.lock composer.lock.original \
   && mv vendor vendor.original \
   && composer require --update-with-all-dependencies --dev \
       cweagans/composer-patches \
-      behat/mink-selenium2-driver:1.3.x-dev \
```

Comment:
Removing `behat/mink-selenium2-driver` and `drupal/coder` b/c they are included in `drupal/core-dev`.